index.html

```diff
@@ -0,0 +1,188 @@
+                <ul class="header__nav">
+                    <li class="nav__item">The Team</li>
+                    <li class="nav__item">Pricing</li>
+                    <li class="nav__item">Features</li>
+                </ul>
+            </nav>
+        </div>
+        <div class="header__container--second container">
+            <div class="header__content1 cell"> 
+                <h2 class="upper--bold">Beautiful Free Nova template</h2>
+                <p class="lower">A top notch premium quality template for your next web project.</p>
+                <button class="button">Download FREE!</button>
+
+            </div>
+            <div class="header__content2 cell"> 
+                <img class="screen" src="./images/screen.png" alt="screen">
+            </div>
+           
+           
+        </div>
+    </header>
+
+    <section class="section1 row">
+        <div class="section1__container--first container">FEATURES</div>
+        <div class="section1__container--second container">
+            <div class="container__second1 cell">
+                <div class="box">
+                    <img src="./images/responsive.svg" alt="responsive">
+                </div>
+                <div class="name1">FULLY RESPONSIVE</div>
+                <div class="description1">Looks amazing on any device: smartphone, tablet, laptop and desktop</div>
+            </div>
+            <div class="container__second1 cell">
+                <div class="box">
+                    <img src="./images/customizable.svg" alt="customizable">
+                </div>
+                <div class="name1">CUSTOMIZABLE</div>
+                <div class="description1">Change the color, pictures or any of the sections to suit your needs</div>
+            </div>
+            <div class="container__second1 cell">
+                <div class="box">
+                    <img src="./images/design.svg" alt="design">
+                </div>
+                <div class="name1">SLICK AND BEAUTIFUL DESIGN</div>
+                <div class="description1">Trendy and fresh design, fits any website</div>
+            </div>
+        </div>
+    </section>
+
+    <section class="section2 row">
+        <div class="section2__container container">
+            <img src="./images/logos.png" alt="logos">
+        </div>
+    </section>
+
+    <section class="section3 row">
+        <div class="content1 cell">
+            <img class="imac" src="./images/imac.png" alt="imac">
+        </div>
+        <div class="content2 cell">
+            <h2 class="upper--normal">SIMPLE AND BEAUTIFUL</h2>
+            <p class="lower">Use Nova theme for your next web project. it is completely customizable so you can change any of the sections to fit your needs. Nova Theme is Free for any kind of use, personal and commercial. Have fun and good luck!</p>
+        </div>
+    </section>
+
+    <section class="section4 row">
+        <div class="section4__container--first4 container">
```

Comment:
Skoro nagłówek to może po prostu `<header>` - ten znacznik może być użyty jako nagłówek dla całej strony, ale również sekcji czy artykułu.